src/assets/spinner.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" stroke="currentColor" viewBox="-2 -2 42 42"><g fill="none" fillRule="evenodd"><g stroke-width="6" transform="translate(1 1)"><circle cx="18" cy="18" r="18" stroke-opacity=".5"/><path d="M36 18c0-9.94-8.06-18-18-18"><animateTransform attributeName="transform" dur="1s" from="0 18 18" repeatCount="indefinite" to="360 18 18" type="rotate"/></path></g></g></svg>
\ No newline at end of file
+<svg stroke="currentColor" viewBox="-2 -2 42 42" xmlns="http://www.w3.org/2000/svg"><g transform="translate(1 1)" stroke-width="6" fill="none"><circle cx="18" cy="18" r="18" stroke-opacity=".5"/><path d="m36 18c0-9.94-8.06-18-18-18"><animateTransform attributeName="transform" dur="1s" from="0 18 18" repeatCount="indefinite" to="360 18 18" type="rotate"/></path></g></svg>
```

Comment:
Are you sure the changes here aren't breaking the rendering of this spinner?

Reply:
Yup, I did test the changes 😉

You can also view the rendered SVG on GitHub:
![image](https://user-images.githubusercontent.com/52870424/116952610-0d4aba00-ac59-11eb-9a5d-7901f34a05ff.png)